src/Components/Web/src/Forms/ValidationMessage.cs

```diff
@@ -28,7 +28,7 @@ public class ValidationMessage<TValue> : ComponentBase, IDisposable
         /// <summary>
         /// Specifies the field for which validation messages should be displayed.
         /// </summary>
-        [Parameter] public Expression<Func<TValue>>? For { get; set; }
+        [Parameter(Required = true)] public Expression<Func<TValue>> For { get; set; } = default!;
```

Comment:
Should we set `CurrentEditContext` to be a required cascading parameter as well?